src/frontend/src/lib/components/dapps/DappPromoBanner.svelte

```diff
@@ -0,0 +1,39 @@
+<script lang="ts">
+	import Button from '$lib/components/ui/Button.svelte';
+	import Img from '$lib/components/ui/Img.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import type { DappDescription } from '$lib/types/dappDescription';
+	import { replacePlaceholders } from '$lib/utils/i18n.utils';
+
+	export let dAppDescription: Omit<DappDescription, 'screenshots'> &
+		Required<Pick<DappDescription, 'screenshots'>>;
+</script>
+
+<article class="relative flex h-64 items-end overflow-hidden rounded-2xl">
+	{#if dAppDescription.screenshots.length > 0}
+		<div class="absolute">
```

Comment:
Just a note for myself: we probably also gonna need to review this and we might need an `object-fit: cover;` but let's move on.